multimodal/vl2l/src/mlperf_inference_multimodal_vl2l/cli.py

```diff
@@ -102,6 +103,34 @@ class TestSettings(BaseModel):
         ),
     ] = 100
 
+    server_expected_qps: Annotated[
+        float,
+        Field(
+            description="The expected QPS for the server scenario.",
+        ),
+    ] = 1
+
+    server_target_latency: Annotated[
+        float,
+        Field(description="Expected latency for Server scenario "
```

Comment:
We should specify in the description how `server_target_latency` works together with `server_expected_qps`.